readthedocs/config/config.py

```diff
@@ -114,12 +114,20 @@ def __init__(self, key, code, error_message, source_file=None):
         self.code = code
         self.source_file = source_file
         message = self.message_template.format(
-            key=key,
+            key=self._get_display_key(key),
             code=code,
             error=error_message,
         )
         super().__init__(message, code=code)
 
+    @staticmethod
```

Comment:
Any reason to make this a static method? Looks like a normal method works

Reply:
@stsewd if I use `instance` methods then the lint fails with `no-self-use` error. So, I used `staticmethod`. Though we can use `#pylint disable=no-self-use`

Reply:
You can use self now `self.key`

Reply:
Done!